pyincore/analyses/housingrecovery/housingrecovery.py

```diff
@@ -128,22 +122,15 @@ def run(self):
         addl_structure_info = self.get_input_dataset("building_area").get_dataframe_from_csv(low_memory=False)
         bg_mhhinc = self.get_input_dataset("census_block_groups_data").get_dataframe_from_csv(low_memory=False)
 
-        # Census API using pyincore_data method
-        columns_api = "GEO_ID,NAME,B25002_001E,B25002_001M,B25004_006E,B25004_006M"
-        fips = self.get_parameter("fips")
-        if fips is None:
-            if self.get_input_dataset("census_appraisal_data") is None:
+        # Census data
+        if self.get_input_dataset("census_appraisal_data") is None:
                 sys.exit("Census data is required!.")
```

Comment:
I don't think either of the sys.exits are needed. It's a required input so it should fail earlier with missing required input dataset.